src/EnergyPlus/PluginManager.cc

```diff
@@ -404,6 +404,14 @@ void PluginManager::setupOutputVariables([[maybe_unused]] EnergyPlusData &state)
 
 PluginManager::PluginManager(EnergyPlusData &state) : eplusRunningViaPythonAPI(state.dataPluginManager->eplusRunningViaPythonAPI)
 {
+    // Now read all the actual plugins and interpret them
+    // IMPORTANT -- DO NOT CALL setup() UNTIL ALL INSTANCES ARE DONE
+    std::string const sPlugins = "PythonPlugin:Instance";
+    int pluginInstances = state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, sPlugins);
+    if (pluginInstances == 0) {
+        return;
+    }
```

Comment:
Don't try to instantiate a plugin manager if there are no PythonPlugin:Instances